storage-resize-images/extension.yaml

```diff
@@ -304,6 +314,28 @@ params:
     required: true
     immutable: false
 
+  - param: DO_BACKFILL
+    label: Backfill existing images
+    description: >
+      Should existing, unresized images in the Storage bucket be resized as well?
+    type: select
+    options:
+      - label: Yes
+        value: true
+      - label: No
+        value: false
+
 events:
   - type: firebase.extensions.storage-resize-images.v1.complete
     description: Occurs when image resizing completes. The event will contain further details about specific formats and sizes.
+
+lifecycleEvents:
+  onInstall:
+    function: backfillResizedImages
+    processingMessage: Resizing existing images in ${param:IMG_BUCKET}
```

Comment:
Has this been tested? I don't remember if we look for variables in the processing message... (but we should)

Reply:
This should theoretically work now, but double check to make sure.

Reply:
Tested and working ok 👍🏻